<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>TopView</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>431</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>591</width>
        <height>431</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <row>
       <property name="text">
        <string>PC name</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>launch time</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>running time</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>system version</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>CPU type</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>CPU HZ</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>content</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Process</string>
     </attribute>
     <widget class="QTableWidget" name="table_proc">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>431</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>pre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>memUsed</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_pNum">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pRun">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>170</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pSleep">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="proc_creat">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>create proc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="proc_shut">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>shut proc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pZombie">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Edit_pid">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Graph</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_now">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cpu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU_USAGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_memory">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MEM_USAGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_shut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>490</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>shut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reboot">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reboot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
